doc/getting-started.rst

```diff
@@ -97,15 +98,25 @@ Install the AArch64 toolchain.
 Ubuntu 22.04
 ------------
 
-Support for Ubuntu 22.04 is still experimental.
+This is fully supported. This is recommended for the `RISC-V` host target. gcc
+version 11 is recommended for this operating system.
+
+Ubuntu 24.04
+------------
+
+This is supported, but has not been tested to the same level as 22.04. gcc
+version 13 is recommended for this operating system. This will be required for
+some qemu RISC-V features such as the half extension which was added in version
+7.
 
 Windows 10
 ----------
 
 On an x86-64 Windows 10 host system these are the minimum requirements to build
-the oneAPI Construction Kit. These tools are required for any build configuration.
+the oneAPI Construction Kit. These tools are required for any build
+configuration. Windows 11 has not been tested but may work.
 
-* `Visual Studio <https://www.visualstudio.com/>`_ 2017 or 2019
+* `Visual Studio <https://www.visualstudio.com/>`_ 2017 or 2019. 2022 may work but is not tested.
```

Comment:
It turns out Visual Studio 2017 doesn't work. The Visual Studio CMake generators don't work with any version of Visual Studio, we assume the build configuration is known at CMake time which it isn't with the VS CMake generator, and with the Ninja CMake generator, we get an error because VS2017 comes with a too old version of Ninja. We should update the instructions to use Ninja everywhere, and VS2022 works fine (except in debug builds, but this is not specific to VS2022; I'll create a PR for that).